sponsorship_compassion/models/contracts.py

```diff
@@ -592,7 +592,7 @@ def unlink(self):
                 if contract.child_id.sponsor_id == contract.correspondent_id:
                     child = contract.child_id.with_context({})
                     child.child_unsponsored()
-                    child.child_released()
+                    child.child_released('N')
```

Comment:
I think the call should simply be removed, what do you think?
```suggestion
```

Reply:
After a few test it seems removing the line doesn't work properly. 
- The hold is in the expected state ("Consignment Hold") but is not available for new sponsorships. 
- The sponsorship is fully deleted. It doesn't appear of the tree view as a cancelled sponsorship (which was the case before)

Reply:
Ok but I fear that we will lose the link from the child to the hold because of this line: https://github.com/CompassionCH/compassion-modules/blob/be41600918ccdcb383f8b07b83e61953ee952b14/child_compassion/models/child_compassion.py#L674
The `hold_id` shouldn't be set to False in this case.